pkg/controller/render/hash.go

```diff
@@ -20,7 +20,7 @@ var (
 	}
 )
 
-func getMachineConfigHashedName(config *mcfgv1.MachineConfig) (string, error) {
+func getMachineConfigHashedName(prefix string, config *mcfgv1.MachineConfig) (string, error) {
```

Comment:
the name actually becomes `prefix-hash`, add a comment on function and what about empty prefix.

Reply:
OK, I tweaked this to just take the pool.